src/main/java/pl/patrycja/ox/gameflow/AutomaticTests.java

```diff
@@ -0,0 +1,82 @@
+package pl.patrycja.ox.gameflow;
+
+import pl.patrycja.ox.GameSettings;
+import pl.patrycja.ox.Sign;
+import pl.patrycja.ox.ui.UI;
+
+import java.util.List;
+
+class AutomaticTests extends Mode {
+
+    AutomaticTests(UI ui) {
+        super(ui);
+    }
+
+    @Override
+    public List<Player> createPlayers() {
+        Player firstPlayer = new Player("Henio", Sign.CROSS);
+        Player secondPlayer = new Player("Olaf", Sign.NAUGHT);
+        return List.of(firstPlayer, secondPlayer);
+    }
+
+    @Override
+    public void settings(String[] inputArrayParameters) {
+        gameSettings = setup(inputArrayParameters);
+        Sequence sequence = new Sequence(gameSettings, inputArrayParameters[2]);
+        sequence.generateSequence();
+    }
+
+    private GameSettings setup(String[] inputArrayParameters) {
+        checkCorrectInputData(inputArrayParameters);
+        //TODO: generate draw sequence!!
```

Comment:
Should this be here? Where are you generating other sequences?

Reply:
Yes, good remark. I deleted the method generating sequence but I forgot to change the place for the todo.